python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -164,6 +164,12 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
         doc="Task to combine two snaps to make one exposure.",
     )
 
+    do_normalized_calibration = pexConfig.Field(
```

Comment:
Is there a reason we wouldn't want to use compensated tophats here? I've avoided `do_x` configs in CalibrateImage, opting to instead just implement the algorithm that we want for each step. If this is something for which there's no real reason to turn it off, there's no need to have a `do_x` for it. That would remove a lot of the branches in the code.

Reply:
This is one of the things I wanted to ask you about.  If you think that it should not be optional, then I can clean this up.

Reply:
Is there any reason you think it should be optional? If not, then I see no benefit to making it so.

Reply:
I'm willing to stand behind the new calibration fluxes, so I'll remove the optionality.